frontend/elements/src/core/menu/kebab.ts

```diff
@@ -41,30 +41,68 @@ export class _ extends LitElement {
     @property({ type: Number })
     offsetHorizontal: number = 50; //enough for a scrollbar
 
-    onGlobalMouseDown = (evt: MouseEvent) => {
-        const path = evt.composedPath();
-        if (
-            !path.includes(
-                this.shadowRoot?.getElementById("menu-container") as any
-            ) &&
-            !path.includes(this.shadowRoot?.getElementById("button") as any)
-        ) {
-            this.visible = false;
-            this.dispatchEvent(new CustomEvent("close"));
+    /// Work-around property so that implementations of kebab which use OverlayHandle and the
+    /// <menu-items /> element can make use of menu-items positioning instead of letting this
+    /// element do work and potentially causing scrolling issues on views which already have
+    /// positioning such as module/edit views.
+    @property({ type: Boolean })
+    positioningEnabled: boolean = true;
+
+    /// Whenever this element needs to be used with some other container outside of this element,
+    /// the user can set this property so that clicking in it doesn't dispatch any events.
+    @property()
+    customContainer?: HTMLElement;
+
+    /// Set the event listener so that it can be reliably removed later.
+    eventListener: any;
```

Comment:
Important additions so that users of this element can pass in custom containers, notably anything rendered with `OverlayHandle`.